src/shared/components/card/CardImage.tsx

```
@@ -0,0 +1,54 @@
+  const visualState = disabled ? 'disabled' : state;
+
+  return (
+    <div
+      className={styles.cardcontainer({ state: visualState })}
+      onMouseDown={handleMouseDown}
+      onMouseUp={handleMouseUp}
+    >
+      <img
+        src={example}
+        className={disabled ? styles.disabledcardimg : styles.cardimg}
+      />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)